python/py-smmap/Portfile

```diff
@@ -1,24 +1,22 @@
-# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem        1.0
 
-PortSystem          1.0
+PortGroup         python 1.0
+PortGroup         github 1.0
 
-PortGroup           python 1.0
-PortGroup           github 1.0
+github.setup      gitpython-developers smmap 3.0.4 v
+name              py-smmap
+maintainers       nomaintainer
+platforms         darwin
+license           BSD
 
-github.setup        gitpython-developers smmap 2.0.5 v
-name                py-smmap
-maintainers         nomaintainer
-platforms           darwin
-license             BSD
+description       Pure python sliding memory map manager
+long_description  ${description}
 
-description         Pure python sliding memory map manager
-long_description    ${description}
+checksums           rmd160  2c46988e9e422f79cf52376ae131ad34efd0f7d6 \
+                    sha256  48491857fe12b7aa59504ea9efba07b056214fbc9380b8c2aa1eda1d76d1dd1f \
+                    size    29227
 
-checksums           rmd160  a5613971d57514243c9e1c4e1560290e709a851e \
-                    sha256  bc94d885e63934dbeb80cc5131c63ab3a920749d22386234757818d516e50db4 \
-                    size    28555
-
-python.versions     27 35 36 37
+python.versions   35 36 37
 python.default_version 36
```

Comment:
can we add `38` to `python.versions` as well? Additionally, it would be best not to set `python.default_version` and just let the python PG take care of this. Currently, Python 3.8 is the default.

Reply:
also, the py27-subport is still a dependent:
```
> port echo depends:py27-smmap
py27-gitdb
```